README.adoc

```diff
@@ -1,11 +1,9 @@
-= Address Book (Level 4)
+= Know-It-All
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
-https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
+https://travis-ci.com/cs2103-ay1819s2-w10-4/main[image:https://travis-ci.com/cs2103-ay1819s2-w10-4/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/dlqs/main[image:https://ci.appveyor.com/api/projects/status/w7fr5fgsrmi981o6?svg=true[Build status]]
```

Comment:
Btw, i thought travis and appveyor ar both CI tools, why do we need to use both ?

Reply:
Travis CI is a CI tool: makes sure that our project can be built (without errors)
Appveyor is a CI tool, but for Windows: makes sure that our project can be build on Windows (without errors)
Coveralls is a code coverage tool: tells us how much our tests cover our code (%LoC tested).